alacritty/src/display/mod.rs

```diff
@@ -484,6 +484,10 @@ impl Display {
 
         window.set_visible(true);
 
+        // Automatically focus new windows, even if no Alacritty windows are currently focused.
```

Comment:
```suggestion
        // Always focus new windows, even if no Alacritty window is currently focused.
```

I know I suggested this myself, but that's just garbage English.